lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -204,16 +208,17 @@ def add_tunnel_flows(self, precedence:int, i_teid:int,
                                priority=priority, goto_table=self.next_table)
 
             # Add flow for mtr port
-            match = MagmaMatch(eth_type=ether_types.ETH_TYPE_IP,
-                               in_port=self.config.mtr_port,
-                               ipv4_dst=ue_ip_adr)
+            match = MagmaMatch(eth_type=get_eth_type(ue_ip_adr),
+                               in_port=self.config.mtr_port, **ip_match_out)
 
             flows.add_flow(self._datapath, self.tbl_num, match, actions=actions,
```

Comment:
sorry for not noticing this in earlier reviews but can we just use the add_resubmit_next_service_flow? In that case its clear that its going to another table

Reply:
Changes done. Removed "flows.add_flow" and using flows.add_resubmit_next_service_flow with introduced new flag "reset_default_register".